x/ethbridge/client/cli/metadata.go

```diff
@@ -36,3 +43,90 @@ func GetCmdGetTokenMetadata() *cobra.Command {
 		},
 	}
 }
+
+// GetCmdSetCrossChainFee is the CLI command to send the message to set crosschain fee for network
+func GetCmdAddIBCTokenMetadata() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "metadata-add [cosmos-sender-address] [token-name] [token-symbol] [token-address] [token-decimals] [network-descriptor]",
+		Short: "Used to manually add Token Metadata for IBC tokens.",
+		Args:  cobra.ExactArgs(6),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			cosmosSender, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			tokenName := args[1]
+
+			tokenSymbol := args[2]
```

Comment:
We should probably check both tokenName and tokenSymbol to ensure the strings are not empty.